packages/extensions/openapi-to-typespec/src/emiters/emit-client.ts

```diff
@@ -34,8 +38,14 @@ function generateClient(program: TypespecProgram) {
         .filter((r) => r !== "")
         .join("\n\n")
     : "";
+
+  const enums = models.enums
+    .map(generateEnumClientDecorator)
+    .filter((r) => r !== "")
+    .join("\n\n");
+
   if (objects === "" && armResources === "") {
```

Comment:
```suggestion
  if (objects === "" && armResources === "" && enums === "") {
```

need also check enums should be empty if there is no renaming,

Reply:
fixed.thx